packages/renderer/src/components/Modals/PublishProject/PublishToWorld/styles.css

```diff
@@ -0,0 +1,100 @@
+/* Select World Modal */
+.MuiModal-root .SelectWorld {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  gap: 24px;
+  padding: 0 24px;
+}
+
+.MuiModal-root .SelectWorld .box {
+  display: flex;
+  width: 100%;
+  gap: 24px;
+}
+
+.MuiModal-root .SelectWorld .thumbnail {
+  width: 250px;
```

Comment:
I see the image a bit stretched:

<img width="870" alt="Screenshot 2024-09-30 at 12 36 05 PM" src="https://github.com/user-attachments/assets/db40ea46-1ba3-419b-baa4-6caef513913a">

Maybe this could be improved to make it cover instead of fit:

<img width="702" alt="Screenshot 2024-09-30 at 12 38 38 PM" src="https://github.com/user-attachments/assets/feb951a4-9871-45bb-abe5-43e08f4003cf">

```suggestion
  width: 250px;
  object-fit: cover;
```

Reply:
thanks for the review! yeah, it looks better making it a cover.. updated! 🙌